tests/integration/tests/namespace/test-protected-resources.spec.js

```diff
@@ -127,6 +127,11 @@ context('Test Protected Resources', () => {
   it('Change protection setting', () => {
     cy.get('[title="Profile"]').click();
 
+    cy.get('ui5-li:visible')
+      .contains('Preferences')
+      .invoke('show')
```

Comment:
here as well

Reply:
Form me without `.invoke('show')` it couldn't find `li`